src/database/migrate.ts

```diff
@@ -1,13 +1,14 @@
 import path from 'path';
-import { migrate as pgMigrate } from 'postgres-migrations';
+import { migrate as pgMigrate } from 'postgres-schema-migrations';
 import { db } from './db';
 
-export const migrate = async (): Promise<void> => {
+export const migrate = async (schema = 'public'): Promise<void> => {
```

Comment:
Someone with stronger DB opinions should let me know if it would be better to use a `pdc` schema as our default, rather than `public.

Reply:
I think it's OK to default to `public`. My sense is that more often than not a dedicated database instance gets created for each separate application.